cumulusci/tasks/bulkdata/mapping_parser.py

```diff
@@ -0,0 +1,34 @@
+from typing import Dict, List
+
+from pydantic import Field
+
+from cumulusci.utils.yaml.model_parser import CCIDictModel
+
+
+class Lookup(CCIDictModel):
+    table: str
+    key_field: str = None
+    value_field: str = None
+    join_field: str = None
+    after: str = None
+    aliased_table: str = None
+
+
+class Step(CCIDictModel):
+    sf_object: str
+    table: str = None
+    fields_: Dict[str, str] = Field(..., alias="fields")
+    lookups: Dict[str, Lookup] = {}
+    static: Dict[str, str] = {}
+    filters: List[str] = []
+    action: str = "insert"
+    oid_as_pk: bool = False  # this one should be discussed and probably deprecated
```

Comment:
I got the impression from the code that it can be inferred instead of declared:

https://github.com/SFDO-Tooling/CumulusCI/blob/d81c3e90d424e57db8d1fc2add7a98b8f480e27a/cumulusci/tasks/bulkdata/utils.py#L176

Reply:
d599001a18913aa